RELEASE-NOTES.txt

```diff
@@ -8,6 +8,7 @@
 * [***] Site Comments: Updated comment details with a fresh new look and capability to display rich contents. [#17466]
 * [**] Block editor: Image block: Add ability to quickly link images to Media Files and Attachment Pages [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3971]
 * [**] Block editor: Fixed a crash that could occur when copying lists from Microsoft Word. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4174]
+* [***] Fixed crash where uploading image when offline crashes iOS app. [#17488]
```

Comment:
Quite the redundant wording: "a crash which crashes" 😅 

I'll let @mokagio decide but imho we shouldn't update the AppStore, Editorial'd release notes for this new entry at that point, especially so far after the code-freeze and as the release notes items from this list have already been thru Editorial edit, and their translations have already started.

This is even more true this time around, because we've _already_ updated the Editorial'd release notes for 18.7 _twice_ sooner this week, which invalidated the translations that were already done each time… and was a bit annoying for the translators to see their translations invalidated multiple times in a row right after having submitted them in GlotPress (see the WPorg slack).

Reply:
> Quite the redundant wording: "a crash which crashes"
Agreed, that should be tweaked.

Reply:
I've tweaked and pushed a change @AliSoftware.

Reply:
Makes sense @AliSoftware - thanks.